src/AddTodoForm.js

```diff
@@ -1,21 +1,27 @@
 import React from 'react';
 
-function AddTodoForm (props) {
+function AddTodoForm ({onAddTodo}) {
 
-    function handleAddTodo(event) {
-        event.preventDefault();
-        const todoTitle = event.target.title.value;
+    const [todoTitle, setTodoTitle] = React.useState('');
```

Comment:
No need to change this, but since this lesson focuses so much on destructuring I wanted to show you something. You can also destructure imports so you're only bringing in what you need, for example:

```suggestion
import { useState } from "react"; // destructuring useState hook out of React object
...
    const [todoTitle, setTodoTitle] = useState(''); // now you don't need React. in front
```